Bottom: 27c34792f16c41b9b663e90d2335026f9a8ed1cc
Top:    30ad5a2bf3893063fc77f2446a36867fda61c46b
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2008-11-04 17:08:38 -0500

Refresh of jds-lirc-mce2

---

diff --git a/drivers/input/ir/ir-mceusb2.c b/drivers/input/ir/ir-mceusb2.c
index cc13e60..898267a 100644
--- a/drivers/input/ir/ir-mceusb2.c
+++ b/drivers/input/ir/ir-mceusb2.c
@@ -653,6 +653,9 @@ static int usb_remote_probe(struct usb_interface *intf,
 	ret = input_register_device(ir->input);
 	if (ret)
 		goto free_input;
+	ret = input_ir_register(ir->input);
+	if (ret)
+		goto free_input;
 
 	return 0;
